pkg/server/sandbox_run.go

```diff
@@ -498,14 +487,36 @@ func (c *criContainerdService) unmountSandboxFiles(rootDir string, config *runti
 	return nil
 }
 
+// setupPod setups up the network for a pod
+func (c *criContainerdService) setupPod(id string, path string, config *runtime.PodSandboxConfig) (string, error) {
+	labels := map[string]string{
+		"K8S_POD_NAMESPACE":          config.GetMetadata().GetNamespace(),
+		"K8S_POD_NAME":               config.GetMetadata().GetName(),
+		"K8S_POD_INFRA_CONTAINER_ID": id,
+		"IgnoreUnknown":              "1",
```

Comment:
For my education, what does `IgnoreUnknown` mean?

Reply:
not needed. will remove

Reply:
It is needed to ignore the arguments that are not known by the plugin. More details : https://github.com/containernetworking/cni/pull/158 and https://github.com/kubernetes/kubernetes/pull/24983